### PR TITLE
Upgrade protocols detection & Add SPDY support

### PR DESCRIPTION
**Pull request by UnknownErrror**

### CODE REVIEW

File: src/background.js [hunk 1]
```diff
@@ -2,25 +2,32 @@ chrome.runtime.onMessage.addListener((message, sender, sendResponse) => {
...
-    title = "HTTP/2";
-    icon = "h2";
-  } else if (message === "hq") {
+  if (message === "hq") {
     title = "HTTP/2 + QUIC";
     icon = "hq";
   } else if (message.startsWith("h3")) {
+    title = "HTTP/3";
     icon = "h3";
+  } else if (message.startsWith("h2")) { // "h2", "h2c"
```
Comment (pd4d10) [inline src/background.js:18]: Thanks! Is there any documentation for the 'h2c' value? If yes then it would be great to add the doc link as a comment

Comment (UnknownErrror) [reply src/background.js:18]: h2 is HTTP/2 over TLS
h2c is HTTP/2 over TCP
From [www.iana.org/assignments/tls-extensiontype-values/tls-extensiontype-values.xhtml](https://www.iana.org/assignments/tls-extensiontype-values/tls-extensiontype-values.xhtml#alpn-protocol-ids)

Comment (UnknownErrror) [reply src/background.js]: Added

**Review by pd4d10 — APPROVED** [review]

**Comment by pd4d10** [comment]
Thanks!

**Pull request merged**